apps/sms/views/new_message/style/recipients.css

```diff
@@ -79,16 +81,16 @@ Do not remove.
   overflow-y: scroll;
 }
 
+/**
+ * BiDi note: all [contenteditable] blocks must carry a `dir="auto"` attribute,
+ * so we cannot use *-inline-{start,end} properties on them, because the
+ * direction of a dir="auto" element (and therefore the position of start/end)
+ * depends on its content. We want our margin/padding to depend on the document
+ * direction, not element's content.
```

Comment:
note that for margin, logical properties use the element's parent's direction, not the element's direction, which makes a lot more sense.
(this was not working like this when we were using -moz-margin-start/end, but they changed it for margin-inline-start/end).

See https://drafts.csswg.org/css-logical-props/#propdef-margin-inline-start. I'm quite sure Gecko implements this.

So maybe we can use the logical props after all ?